test/unit/middleware/validateSurnameTest.js

```diff
@@ -0,0 +1,74 @@
+const { validateSurname } = require('app/middleware/validateSurname');
+const { sinon } = require('test/chai-sinon');
+const validateSurnameContent = require('app/assets/locale/en').validateSurname;
+
+describe('validateSurname.js', () => {
+
+  const mactoken = 'validToken';
+  let req, res, next;
+
+  beforeEach(() => {
+    req = {
+      params: {
+        mactoken
+      },
+      body: {}
+    };
+    res = {
+      render: sinon.stub(),
+      locals: {
+        i18n: {
+          validateSurname: validateSurnameContent
+        }
+      }
+    };
+    next = sinon.stub();
+  });
+
+  describe('no field errors', ()=> {
+
+    it('calls next when there are no field errors', () => {
+      req.body.surname = 'validSurname';
+      validateSurname(req, res, next);
+      next.should.have.been.called;
```

Comment:
What happened to `expect(..)`? For example: `expect(myStub).to.have.been.calledOnce;`